app/controllers/close_contacts_controller.rb

```diff
@@ -57,10 +57,10 @@ def destroy
       reason = params.permit(:delete_reason)[:delete_reason]
       History.close_contact_edit(patient: params.permit(:patient_id)[:patient_id],
                                  created_by: current_user.email,
-                                 comment: "User deleted a close contact (ID: #{@close_contact.id}, Name: #{@close_contact.first_name} #{@close_contact.last_name}"\
-                                 ", Primary Telephone: #{@close_contact.primary_telephone}, Email: #{@close_contact.email}, Last Date of Exposure: "\
-                                 "#{@close_contact.last_date_of_exposure}, Assigned User: #{@close_contact.assigned_user}, Notes: #{@close_contact.notes}, "\
-                                 "Contact Attempts: #{@close_contact.contact_attempts}, Reason: #{reason}.)")
+                                 comment: "User deleted a close contact (ID: #{@close_contact.id}, Name: #{@close_contact.first_name} "\
```

Comment:
this looks different than it does for vaccines and labs.  can you only show the non-blank elements and pull the reason out of the parenthesis? like this:

```
        reason = params.permit(:delete_reason)[:delete_reason]
        comment = "User deleted a lab result (ID: #{@lab.id}"
        comment += ", Type: #{@lab.lab_type}" unless @lab.lab_type.blank?
        comment += ", Specimen Collected: #{@lab.specimen_collection}" unless @lab.specimen_collection.blank?
        comment += ", Report: #{@lab.report}" unless @lab.report.blank?
        comment += ", Result: #{@lab.result}" unless @lab.result.blank?
        comment += "). Reason: #{reason}."
```